src/_nebari/upgrade.py

```diff
@@ -678,15 +723,21 @@ class Upgrade_2023_7_2(UpgradeStep):
 
     @override
     def _version_specific_upgrade(
-        self, config, start_version, config_filename: Path, *args, **kwargs
+        self,
+        config,
+        start_version,
+        config_filename: Path,
+        *args,
+        attempt_fixes=False,
```

Comment:
Same thing as above: `attempt_fixes` should already be exposed in `kwargs`

Reply:
Fixed